app/uk/gov/hmrc/helptosavefrontend/auth/HelpToSaveAuth.scala

```diff
@@ -120,6 +123,14 @@ trait HelpToSaveAuth extends AuthorisedFunctions with AuthRedirects with Logging
       }
     }
 
+  private def isInMaintaince(currantTime: Long, maintainceTimes: Seq[appConfig.MaintainceTimes]): Boolean = {
```

Comment:
Spelling, this may have been Colm, should be currentTime, not when we eat raisins :-)